src/ngAria/aria.js

```diff
@@ -364,8 +364,8 @@ ngAriaModule.directive('ngShow', ['$aria', function($aria) {
             elem.attr('tabindex', 0);
           }
 
-          if ($aria.config('bindKeypress') && !attr.ngKeypress) {
-            elem.on('keypress', function(event) {
+          if ($aria.config('bindKeydown') && !attr.ngKeypress) {
```

Comment:
`attr.ngKeypress` should be also changed to `attr.ngKeydown`.
I wonder if it's a good idea to check for both (`ngKeydow` and `ngKeypress`).